com.unity.uiwidgets/Runtime/engine2/UIWidgetsPanel.cs

```diff
@@ -75,8 +102,13 @@ public struct TextFont {
             _handle = GCHandle.Alloc(this);
 
             _ptr = UIWidgetsPanel_constructor((IntPtr) _handle, UIWidgetsPanel_entrypoint);
+            var settings = new Dictionary<string, object>();
```

Comment:
is it better to pass the font settings to engine using another api ? to my personal opinion, we should not give onEnable too much job to do. 

I perfer to add a new api like UIWIdgetsPanel_InitialFontSettings() and call it only when necessary:
```
if (fonts != null && fonts.Length > 0)
{
    //init settings
    //UIWIdgetsPanel_InitialFontSettings(settings);
}
```

Reply:
This data is passed for initialize engine. I think this could be used for store other configs as well.

Reply:
yes you are right :-)